[QUESTION]如何解决@types/superagent版本造成ERR_PNPM_TARBALL_INTEGRITY  Got unexpected checksum for

背景
- 正常midway项目，然后构建
-  出现
```
+ pnpm i --shamefully-hoist
Progress: resolved 1, reused 0, downloaded 0, added 0
Packages: +889
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Packages are hard linked from the content-addressable store to the virtual store.
  Content-addressable store is at: /var/lib/jenkins/.local/share/pnpm/store/v3
  Virtual store is at:             node_modules/.pnpm
Progress: resolved 889, reused 22, downloaded 0, added 6
 WARN  GET https://registry.npmmirror.com/@types/superagent/-/superagent-4.1.14.tgz error (ERR_PNPM_TARBALL_INTEGRITY). Will retry in 10 seconds. 2 retries left.
Progress: resolved 889, reused 193, downloaded 1, added 171
Progress: resolved 889, reused 521, downloaded 1, added 445
Progress: resolved 889, reused 727, downloaded 1, added 723
Progress: resolved 889, reused 877, downloaded 1, added 885
Progress: resolved 889, reused 879, downloaded 1, added 887
Progress: resolved 889, reused 879, downloaded 1, added 888
 WARN  GET https://registry.npmmirror.com/@types/superagent/-/superagent-4.1.14.tgz error (ERR_PNPM_TARBALL_INTEGRITY). Will retry in 1 minute. 1 retries left.
 ERR_PNPM_TARBALL_INTEGRITY  Got unexpected checksum for "https://registry.npmmirror.com/@types/superagent/-/superagent-4.1.14.tgz". Wanted "sha512-mu/N4uvfDN2zVQQ5AYJI/g4qxn2bHB6521t1UuH09ShNWjebTqN0ZFuYK9uYjcgmI0dTQEs+Owi1EO6U0OkOZQ==". Got "sha512-iiXaOL2wSbnSY4qg0mFPWJHL9iwyEsoNYwaHF2w58/fsVAQJlj+KUfFAFZu+nzbz+b7dUprJEAc+O9vhHHhQTA==".
Build step 'Execute shell' marked build as failure
Finished: FAILURE
```

这个包暂时是@midway/mock里有引用到 然后还像这个包也建议用4.1.14
```
  /@types/superagent/4.1.15:
    resolution: {integrity: sha512-mu/N4uvfDN2zVQQ5AYJI/g4qxn2bHB6521t1UuH09ShNWjebTqN0ZFuYK9uYjcgmI0dTQEs+Owi1EO6U0OkOZQ==, tarball: '@types/superagent/-/superagent-4.1.14.tgz'}
    deprecated: '[WARNING] Use 4.1.14 instead of 4.1.15, reason: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/58445'
    dependencies:
      '@types/cookiejar': 2.1.2
      '@types/node': 14.18.23
    dev: true
```

--- a/packages/mock/package.json
+++ b/packages/mock/package.json
@@ -40,6 +40,7 @@
   "dependencies": {
     "@midwayjs/async-hooks-context-manager": "^3.4.7",
     "@types/supertest": "2.0.12",
+    "@types/superagent": "4.1.14",
     "fs-extra": "10.0.1",
     "js-yaml": "4.1.0",
     "raw-body": "2.5.1",

